docs/keycodes.md

```diff
@@ -191,6 +191,8 @@ This is a reference only. Each group of keys links to the page documenting their
 |`KC_MEDIA_PLAY_PAUSE`  |`KC_MPLY`           |Play/Pause Track                               |
 |`KC_MEDIA_SELECT`      |`KC_MSEL`           |Launch Media Player (Windows)                  |
 |`KC_MEDIA_EJECT`       |`KC_EJCT`           |Eject (macOS)                                  |
+|`KC_SCREEN_BRIGHTNESS_UP`   | `KC_SBRU`       |Increase screen brightness                   |
```

Comment:
```suggestion
|`KC_BRIGHTNESS_UP`     |`KC_BRUP`           |Increase screen brightness                     |
```
I suggest removing the `SCREEN_` part. It seems superfluous, makes the full keycode name too long, and it's not a standardized name in the HID specification anyway.